ament_lint_common/ament_lint_common-extras.cmake.in

```diff
@@ -0,0 +1,6 @@
+# generated from ament_lint_common/ament_lint_common-extras.cmake.in
+
+set(_exec_depends "@ament_lint_common_EXEC_DEPENDS@")
+foreach(_dep ${_exec_depends})
+  find_package(${_dep} REQUIRED)
+endforeach()
```

Comment:
Good point. I think at some point I did more debugging in here. 8f269028f22aa5cf59a3325bda8666abc7e436f7 changes it to export the dependencies without a custom extras file.